modules/home-manager/gtk.nix

```diff
@@ -70,23 +70,14 @@ in
     ];
 
     gtk = {
-      theme =
-        let
-          flavorUpper = ctp.mkUpper cfg.flavor;
-          accentUpper = ctp.mkUpper cfg.accent;
-          sizeUpper = ctp.mkUpper cfg.size;
-
-          # use the light gtk theme for latte
-          gtkTheme = if cfg.flavor == "latte" then "Light" else "Dark";
-        in
-        {
-          name = "Catppuccin-${flavorUpper}-${sizeUpper}-${accentUpper}-${gtkTheme}";
-          package = pkgs.catppuccin-gtk.override {
-            inherit (cfg) size tweaks;
-            accents = [ cfg.accent ];
-            variant = cfg.flavor;
-          };
+      theme = {
+        name = "catppuccin-${cfg.flavor}-${cfg.accent}-${cfg.size}+default";
```

Comment:
i believe tweaks are added at the end. For example `+default` for no tweaks, `+black` for the black tweak.

Reply:
So the tweak to path mapping is this?
- black -> +black
- rimless -> +rimless
- normal -> +default

Reply:
seem correct